lib/pipeline/statelessPipelineStack.ts

```diff
@@ -20,6 +20,41 @@ export class StatelessPipelineStack extends cdk.Stack {
   constructor(scope: Construct, id: string, props: cdk.StackProps) {
     super(scope, id, props);
 
+    // Define CodeBuild project for GH action runner to use
+    // the GH repo defined below already configured to allow CB webhook
+    // This is actually not part of the pipeline, so I guess we could move this someday.
+    const ghRunnerRole = new iam.Role(this, 'GHRunnerRole', {
+      assumedBy: new iam.ServicePrincipal('codebuild.amazonaws.com'),
+    });
+    new codebuild.CfnProject(this, 'GHRunnerCodeBuildProject', {
+      // the name here act as a unique id for GH action to know which CodeBuild to use
+      // So if you change this, you need to update the GH action yaml file (.github/workflows/prbuild.yml)
+      name: 'orcabus-codebuild-gh-runner',
+      description: 'GitHub Action Runner in CodeBuild for `orcabus` repository',
+      serviceRole: ghRunnerRole.roleArn,
+      artifacts: {
+        type: 'NO_ARTIFACTS',
+      },
+      environment: {
+        type: 'ARM_CONTAINER',
+        computeType: 'BUILD_GENERAL1_SMALL',
+        image: 'aws/codebuild/amazonlinux2-aarch64-standard:3.0',
+        privilegedMode: true,
+      },
+      source: {
+        type: 'GITHUB',
+        gitCloneDepth: 1,
+        location: 'https://github.com/umccr/orcabus.git',
+        reportBuildStatus: false,
+      },
+      logsConfig: { cloudWatchLogs: { status: 'DISABLED' } },
```

Comment:
Not seeing anything is useful here, might as well disabled this.

<img width="479" alt="Screenshot 2024-05-13 at 17 42 27" src="https://github.com/umccr/orcabus/assets/61998484/a891881a-7bcc-4813-b641-7373acce1a99">

Reply:
Hm, the useful bits may only come when something goes wrong? 
Are those logs visible via the codebuild history? 
In any case, I guess easy to add back in if we need to...

Reply:
The real testing logs goes to the GH action itself.

https://github.com/umccr/orcabus/actions/runs/9063282042

Reply:
To enable just need to change the `DISABLED` to `ENABLED`, and need to add log group (and its respective role) as this is the L1 construct